extensions/reactive-mysql-client/runtime/src/main/java/io/quarkus/reactive/mysql/client/runtime/DataSourceReactiveMySQLConfig.java

```diff
@@ -38,4 +40,16 @@ public class DataSourceReactiveMySQLConfig {
      */
     @ConfigItem(defaultValueDocumentation = "disabled")
     public Optional<SslMode> sslMode = Optional.empty();
+
+    /**
+     * Connection timeout in seconds
+     */
+    @ConfigItem()
+    public OptionalInt connectionTimeout = OptionalInt.empty();
+
+    /**
+     * Select a different authentication plugin
+     */
+    @ConfigItem
```

Comment:
Can you please add a `defaultValueDocumentation` attribute like we have for `SslMode`?